src/Notification/Container.js

```diff
@@ -0,0 +1,119 @@
+import React, { forwardRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import { Container } from './elements';
+import { Timer } from './helpers';
+
+/**
+ * Notification Container
+ *
+ * This component is in charge of displaying
+ * the notification container
+ *
+ * @param {boolean} autoDismiss - Whether or not to dismiss the notification automatically after a timeout.
+ * @param {number} autoDismissTimeout - The time until a notification is automaticaly dismissed, in milliseconds.
+ * @param {function} component - Presentational component for displaying message.
+ * @param {function} onDismiss - Callback function to call when notification is dismissed.
```

Comment:
You forgot to notify the `onDismiss` callback in the storybook documentation.

Reply:
I don't forget this prop since is part of the <Container /> component